src/components/shop/menu.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * @module components/shop
+ * @copyright BBC 2020
+ * @author BBC Children's D+E
+ * @license Apache-2.0
+ */
+import { getSafeArea } from "./shop-layout.js";
+import { createGelButtons, resizeGelButtons } from "./menu-buttons.js";
+
+export const createMenu = (scene, config) => {
+    const bounds = getSafeArea(scene.layout);
+    const { buttonsRight } = config;
+
+    const menuContainer = scene.add.container();
+    menuContainer.config = config;
+    menuContainer.setVisible = setVisible(menuContainer);
+    menuContainer.resize = resize(menuContainer);
+    menuContainer.memoisedBounds = bounds; // gel button offset bug means we can't rely on getBounds() for now
+
+    menuContainer.add(createRect(scene, bounds, buttonsRight));
+    menuContainer.add(createButtonRect(scene, bounds, !buttonsRight));
+    menuContainer.add(createInnerRect(scene, bounds, buttonsRight));
```

Comment:
```[createRect(scene, bounds, buttonsRight), createButtonRect(scene, bounds, !buttonsRight), createInnerRect(scene, bounds, buttonsRight)].forEach(rect => menuContainer.add(rect));```

dunno if would reduce line usage or not 🤷

Reply:
probably not 😂

Reply:
It didn't, but there was a bit of bagginess in this file, 3e962d0 improves reuse.